docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[ml-nlp-deploy-models]]
+= Import and deploy trained models
+
+:keywords: {ml-init}, {stack}, {nlp}
+:description:  To make trained models available for inference, you must import \
+and deploy them in {es}.
+
+If you want to perform {nlp} tasks in your cluster, you must import and deploy
+an appropriate trained model. There are {es} APIs and tooling support in eland
+to prepare and manage models. 
+
+. <<ml-nlp-select-model,Choose a trained model.>>
+. <<ml-nlp-upload-model,Upload the trained model and vocabulary.>>
+. <<ml-nlp-deploy-model,Deploy the model in your cluster.>>
+
+[discrete]
+[[ml-nlp-select-model]]
+== Select a trained model
+
+Per <<ml-nlp-overview>>, there are multiple ways that you can use NLP features
+within the {stack}. After you determine which type of NLP task you want to
+perform, you must choose an appropriate trained model. You can either create the
+model yourself or find one that meets your needs.
+
+IMPORTANT: The {stack-ml-features} support transformer models that conform to
+the standard BERT model interface and use the WordPiece tokenization algorithm.
+
+The simplest method is to use a model that has already been fine-tuned for the
+type of analysis that you want to perform. For example, there are models and 
+data sets available for specific NLP tasks in
+https://huggingface.co/models[Hugging Face].
+
+[discrete]
+[[ml-nlp-upload-model]]
+== Upload the trained model and vocabulary
+
+After you find or create a model, you must upload it and its tokenizer
+vocabulary to your cluster. 
+
+Trained models must be in a TorchScript representation for use with
+{stack-ml-features}. To use scripts that format and upload the necessary
```

Comment:
Rephrase to something like:
To use scripts that convert existing Hugging Face transformer models to their TorchScript representations, and import them with their tokenizer vocabularies, refer to https://github.com/elastic/eland#nlp-with-pytorch. If you have trained a model yourself using PyTorch directly, you will be responsible for first creating the TorchScript representation of the model. Once you have a TorchScript representation, you can use eland to import the necessary artifacts into Elasticsearch: To use scripts that convert existing Hugging Face transformer models to their TorchScript representations, and import them with their tokenizer vocabularies, refer to https://github.com/elastic/eland#nlp-with-pytorch.

Reply:
Thanks!  I've reworked it in 04a2fce3f59f90c5ee3fdeeef1a2705b187fe070